CHANGELOG.md

```diff
@@ -22,6 +22,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Crash when using unrecognized remote control (#1685)
 - Fixed mismatch between internal and display state after clearing data (#1691)
 
+## [3.3] - 2019-01-31
+*Released to all devices except Fire TV 4K.*  
+  
+Version-bump only - no change from v3.2.5
+
+## [3.2.5] - 2019-01-16
+*Rollback to v3.1.3*
```

Comment:
Can you elaborate a little bit to avoid confusion? So the rollback to v3.1.3 backed out all of 3.2.2, 3.2.1, and v3.2?